configuration/parse.go

```diff
@@ -4,20 +4,27 @@ import (
 	"bufio"
 	"flag"
 	"fmt"
-	"github.com/sonatype-nexus-community/nancy/types"
 	"os"
 	"regexp"
 	"strings"
+
+	"github.com/sonatype-nexus-community/nancy/types"
 )
 
 type Configuration struct {
-	UseStdIn bool
-	Help bool
-	NoColor bool
-	Quiet bool
-	Version bool
-	CveList types.CveListFlag
-	Path    string
+	UseStdIn    bool
+	Help        bool
+	NoColor     bool
+	Quiet       bool
+	Version     bool
+	CveList     types.CveListFlag
+	IQ          bool
+	Path        string
+	User        string
+	Token       string
+	Stage       string
+	Application string
+	Server      string
```

Comment:
At risk of duplicating command line flags, it might make sense to add some docs here?

Reply:
or... maybe an IqConfig struct instead of flattening everything?

Reply:
I like that! Config was completely new to me (I have been away too long!)

Reply:
Did an IQConfig in: https://github.com/sonatype-nexus-community/nancy/pull/68/commits/706eedc0682b1c5132999e07ee536aa9236ebc4f